src/audio/dai-zephyr.c

```diff
@@ -1468,32 +1468,33 @@ int dai_zephyr_multi_endpoint_copy(struct dai_data **dd, struct comp_dev *dev,
 static void set_new_local_buffer(struct dai_data *dd, struct comp_dev *dev)
 {
 	struct comp_buffer __sparse_cache *dma_buf = buffer_acquire(dd->dma_buffer);
+	struct comp_buffer __sparse_cache *local_buf_c;
 	struct comp_buffer __sparse_cache *local_buf;
```

Comment:
ouch, this one isn't `__sparse_cache`, there must be an error in sparse test.

Reply:
Fixed in V2.